src/utils/__tests__/adHelpers.test.js

```diff
@@ -81,10 +81,7 @@ describe('Tab Tracking Methods', () => {
   })
 
   it('returns zero tabs today when the last tab day count is not set in localStorage', () => {
-    localStorageMgr.getItem.mockReturnValueOnce(
-      STORAGE_TABS_LAST_TAB_OPENED_DATE,
-      moment(mockNow).utc().toISOString()
-    )
+    localStorageMgr.getItem.mockReturnValueOnce(undefined)
```

Comment:
Is this related?

Reply:
@jedtan No, but great catch, haha. This was an error that was causing noise in test output that annoyed me.